Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -52,9 +52,16 @@ public class TreeWalkerParameters
 
         /// <summary>
```

Comment:
>/// <summary> [](start = 8, length = 13)

Add a line:
/// [DEPRECATED]

Reply:
fixed

---
In reply to: [504219343](https://github.com/microsoft/Forge/pull/32#discussion_r504219343) [](ancestors = 504219343)